src/client.js

```diff
@@ -17,15 +17,20 @@ const store = configureStore(initialState, baseHistory)
 const history = syncHistoryWithStore(baseHistory, store)
 const root = document.getElementById('app')
 
-const renderApp = () => (
+const renderApp = (renderProps) => (
   <AppContainer>
     <Provider store={store}>
-      <Router history={history} routes={routes} />
+      <Router {...renderProps} />
     </Provider>
   </AppContainer>
 )
```

Comment:
![image](https://cloud.githubusercontent.com/assets/2836593/23390102/f55f05c8-fda6-11e6-9a83-e804a1e886bf.png)

@diegohaz When I don't use the `match` call, the app will successfully render on the client, but it will throw the above warning in console. The reason I added the `match` call was to try and see how I can remove this warning to make sure the client/server rendered pages are the same. My understanding of exactly what is happening with the `render` or `match` calls is not perfect, so would be great to get some input on that.